src/modules/songs/forms/AddSongToSet/components/AddSongToSetDialog.tsx

```
@@ -0,0 +1,103 @@
+    title: "When will you play it?",
+    subtitle: (songName: string) => `Set when you'll play ${songName}`,
+  },
+  [AddSongToSetDialogStep.SET_KEY]: {
+    title: "What key will you play in?",
+  },
+  [AddSongToSetDialogStep.ADD_NOTES]: {
+    title: "Add notes",
+  },
+};
+
+export const AddSongToSetDialog: React.FC = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)